CHANGELOG.md

```diff
@@ -2,6 +2,60 @@
 
 All notable changes to this project are documented in this file.
 
+## 0.21.0
+
+**Release date:** 2022-01-21
+
+This prerelease comes with changes to the base image used to build and
+run the controller, replacing Debian Unstable (Sid) with Debian Bookworm.
+The controller is now statically build and includes libgit2 along with
+its main dependencies.
+
+The controller container images are signed with Cosign and GitHub OIDC
```

Comment:
Would it good to have  a reference to https://github.com/sigstore/cosign/blob/22007e56aee419ae361c9f021869a30e9ae7be03/KEYLESS.md to help people who would be willing to find out more about it ?